tests/lib/rules/no-unnecessary-service-injection-argument.js

```diff
@@ -32,8 +32,6 @@ ruleTester.run('no-unnecessary-service-injection-argument', rule, {
     "const controller = Controller.extend({ serviceName: service('service-name') });",
 
     // Property name does not match service name:
-    "export default Component.extend({ specialName: service('service-name') });",
-    "export default Component.extend({ specialName: inject('service-name') });",
```

Comment:
were these duplicates?

Reply:
Correct!

https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/no-identical-tests.md